src/python2/hangman/hangman.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Hangman
+author: Ole Kristian Pedersen, Kodeklubben Trondheim
+level: 3
+---
+
+# Introduksjon {.intro}
+
+I denne oppgaven skal vi lage vårt eget hangman-spill.  Vi har laget litt
+ferdigskrevet kode for å hjelpe deg på vei. Den kan du [laste ned
+her](./hangman.py). Lagre den der du pleier å lagre Python-koden din.
+
+Det kan se ut som mye kode, men det bare den øverste delen du skal endre på. Et
+lite stykke ned vil det være en overskrift som ser slik ut:
+
+```python
+###############################################################################
+# Library code
+###############################################################################
+```
+
+All koden under denne overskriften kan du bare overse. Dette er koden som
+hjelper deg med å tegne figurer og hente input fra brukeren.
+
+I koden over overskriften har vi deklarert noen variabler:
+
+ * `TRIES` som angir hvor mange forsøk brukeren har. Ettersom det bare er
+   tegnet figurer for 7 feil, så er det lurt å la denne være slik den er.
+
+ * `WORDS` er en liste over ord som kan gjettes, men ingen av disse ordene kan
+   inneholde æ, ø eller å.
```

Comment:
Bug 1.0.x, som nå er fikset i 1.1.x (pga. bruk av enums for å mappe charcode -> bokstav). Jeg kommmer med en oppdatering snart.